app/components/status_tags/heads_of_terms_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module StatusTags
+  class HeadsOfTermsComponent < StatusTags::BaseComponent
+    def initialize(heads_of_term:)
+      @heads_of_term = heads_of_term
+    end
+
+    private
+
+    attr_reader :heads_of_term
+
+    def status
+      if heads_of_term.current_review.present?
+        if heads_of_term.validation_requests.where("validation_requests.created_at >= ?", heads_of_term.current_review.created_at).any? { |validation_request| !validation_request.approved.nil? } && !heads_of_term.current_review.complete?
```

Comment:
Should this be a method or scope in the model?
Seems quite complicated to have here.

I think we were moving away from having the status tag as a component as it's not reusable but maybe for a future clean up